telegram/ext/_defaults.py

```diff
@@ -220,6 +228,21 @@ def run_async(self) -> bool:
     def run_async(self, value: object) -> NoReturn:
         raise AttributeError("You can not assign a new value to run_async after initialization.")
 
+    @property
+    def protect_content(self) -> Optional[bool]:
+        """:obj:`bool`: Optional. Protects the contents of the sent message from forwarding and
+        saving.
+
+        .. versionadded:: 14.0
+        """
+        return self._protect_content
+
+    @protect_content.setter
+    def protect_content(self, value: object) -> NoReturn:
+        raise AttributeError(
+            "You can't assign a new value to protect_content after initialization."
+        )
+
     def __hash__(self) -> int:
```

Comment:
`__hash__` should be updated + probably tests updates for `__hash__` and `__eq__`